QuickDeployment/QuickDeploymentInternetSetup-v1.md

```diff
@@ -146,7 +146,7 @@ Now, you will use `Set-QDEnvironmentInternetSecurity.ps1` to update the scripts
 ```powershell
```

Comment:
Does this page already have the nice messages that this is not as necessary for v2? It was something I requested. But maybe that is a secondary pr.

Reply:
This does not, probably better for a secondary PR. If you'd rather I put them in here though I don't mind :)

Reply:
Want to create an issue so it is not forgotten?

Reply:
Created #212